src/quo2/components/calendar/calendar_year/view.cljs

```diff
@@ -0,0 +1,18 @@
+(ns quo2.components.calendar.calendar-year.view
+  (:require [react-native.core :as rn]
+            [quo2.theme :as theme]
+            [quo2.components.calendar.calendar-year.style :as style]))
+
+(defn- calendar-year-internal
+  [{:keys [selected? disabled? on-press]} year]
+  [rn/touchable-opacity
+   {:on-press on-press
+    :style    (style/container
+               {:selected? selected?
+                :disabled? disabled?})
+    :disabled disabled?}
+   [rn/text
```

Comment:
Probably you don't want to use en/text component here and instead use our component in quo2.components.markdown.text - there should be plenty of examples of this - all of the stylings are already encapsulated there and can be done with the correct configuration 👍